_posts/en/newsletters/2021-12-22-newsletter.md

```diff
@@ -0,0 +1,665 @@
+  help wallet and infrastructure developers verify their
+  implementations.
+
+- **November<!--taproot-->** celebrated [taproot activation][].  There was some
+  initial confusion as the miners of block {{site.trb}} and several
+  subsequent blocks didn't include any taproot-spending transactions.
+  However, thanks to the work of several developers and mining pool
+  administrators, most blocks mined in subsequent days were ready to
+  contain taproot-spending transactions.  Development and [testing][cbf
+  verification] of taproot-ready software [continued][rb#691].
+
+- **December<!--taproot-->** saw Bitcoin Core merge a PR that would
+  allow [descriptor wallets][topic descriptors] to create
+  [bech32m][topic bech32] addresses for receiving taproot payments.  LN
+  devs also further discussed [making use][ln ptlcs] of taproot's
+  features.
+
+Despite complications choosing an activation mechanism for taproot and
+some minor confusion immediately after its activation, the final steps
+of adding support for the taproot soft fork to Bitcoin overall seemed to
+go well.  This is hardly the end the of the story for taproot.  Optech
+expects to continue to spend a significant amount of time writing about
+it in the coming years as wallet and infrastructure developers begin
+making use of its many features.
+
+</div>
+
+## April
+
+LND added [support][lnd#5709] in April for making Atomic Multipath
+Payments (AMP), also called original AMPs due to being described earlier
+than the [Simplified Multipath Payments][topic multipath payments]
+(SMPs) all major LN implementations currently support.  AMPs have a
+privacy advantage over SMPs and also ensure that the receiver has
+received all parts before they claim the payment.  Their downside is
+that they don't produce cryptographic proof of payment.  LND implemented
+them for use with [spontaneous payments][topic spontaneous payments]
+which fundamentally can't provide a proof of payment, eliminating from
+consideration the one significant downside of AMPs.
+
+## May
+
+A discrepancy between the specification of [BIP125][] transaction
+[replacement][topic rbf] and the implementation in Bitcoin Core was
+[disclosed][bip125 discrep] in May.  This didn't put any bitcoins at
+risk, as far as we know, but it did spawn several discussions about the
+risks to contract protocols such as LN from unexpected transaction relay
+behavior.
+
+Also in May, the C-Lightning project [merged][cl#4489] a plugin for
+managing [dual-funded channels][topic dual funding]---channels where
+both parties can provide some amount of the initial funding.  In
+addition to prior dual-funding work [merged][cl#4410] this year, this
+allows the party initiating the channel open to not only spend funds
+through the channel but also receive them in the channel's initial
+state.  Dual-funding is particularly useful for merchants whose primary
+use of LN is receiving payments instead of sending them.
```

Comment:
It's not clear to me why Dual-funding would be helpful for users that primarily receive. Either way, the counterparty has to either open the channel or participate in opening. Perhaps this point could be elaborated.

Reply:
I'll be pushing an edit but I'm not 100% sure it addresses your concern---I'm confused about why you're confused---so let me know if it doesn't help.